extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetailsHeader.ts

```diff
@@ -100,16 +103,22 @@ export class AssessmentDetailsHeader {
 	public async populateAssessmentDetailsHeader(migrationStateModel: MigrationStateModel): Promise<void> {
 		// this value is populated to handle the case when user selects a target type and want to resume later.
 		this._targetSelectionDropdown.value = this.getTargetTypeBasedOnModel(migrationStateModel._targetType);
+
+		const recommendedConfigurations = await utils.getRecommendedConfiguration(migrationStateModel._targetType, migrationStateModel);
+		let configurationValue = recommendedConfigurations[0] ?? "--";
+
+		if (migrationStateModel._targetType === MigrationTargetType.SQLVM && recommendedConfigurations.length > 1) {
+			configurationValue = recommendedConfigurations[0] + "\n" + recommendedConfigurations[1];
+		}
 		const assessmentHeaderValues = [
 			{
-				// TODO(stutijain): replace below value with recommended config
-				value: "10 available"
+				value: configurationValue
 			},
 			{
 				value: String(migrationStateModel?._assessedDatabaseList.length)
 			},
 			{
-				value: String(migrationStateModel._assessmentResults.databaseAssessments.filter((db) => db.issues.filter(issue => issue.appliesToMigrationTargetPlatform === migrationStateModel._targetType).length === 0).length)
+				value: String(migrationStateModel._assessmentResults.databaseAssessments.filter((db) => db.issues.filter(issue => issue.appliesToMigrationTargetPlatform === migrationStateModel._targetType && issue.issueCategory === IssueCategory.Issue).length === 0).length)
```

Comment:
same here, I recommend handling nulls for all *.length references

Reply:
done